Tools/CMake/R.cmake

```diff
@@ -838,14 +838,40 @@ message(STATUS "Setting up renv, Rcpp, RInside, and jaspModuleInstaller")
 message(STATUS "RENV_LIBRARY           = ${RENV_LIBRARY}")
 message(STATUS "R_CPP_INCLUDES_LIBRARY = ${R_CPP_INCLUDES_LIBRARY}")
 
-configure_file(${PROJECT_SOURCE_DIR}/Modules/setup_renv_rcpp_rinside.R.in
-                ${SCRIPT_DIRECTORY}/setup_renv_rcpp_rinside.R @ONLY)
+configure_file(${PROJECT_SOURCE_DIR}/Modules/setup_renv.R.in
+                ${SCRIPT_DIRECTORY}/setup_renv.R @ONLY)
 
 execute_process(
   COMMAND_ECHO STDOUT
   #ERROR_QUIET OUTPUT_QUIET
   WORKING_DIRECTORY ${R_HOME_PATH}
-  COMMAND ${R_EXECUTABLE} --slave --no-restore --no-save --file=${SCRIPT_DIRECTORY}/setup_renv_rcpp_rinside.R)
+  COMMAND ${R_EXECUTABLE} --slave --no-restore --no-save --file=${SCRIPT_DIRECTORY}/setup_renv.R)
+
+if(APPLE)
+  # Patch renv
+  message(CHECK_START "Patching ${RENV_LIBRARY}")
+  execute_process(
+    COMMAND_ECHO STDOUT
+    #ERROR_QUIET OUTPUT_QUIET
+    WORKING_DIRECTORY ${R_HOME_PATH}
+    COMMAND
+      ${CMAKE_COMMAND} -D
+      NAME_TOOL_PREFIX_PATCHER=${PROJECT_SOURCE_DIR}/Tools/macOS/install_name_prefix_tool.sh
```

Comment:
@RensDofferhoff just for my understanding, which .so file needed to get patched here? Something from Rcpp of RInside?

Reply:
See the renv changelog: https://rstudio.github.io/renv/news/index.html

There now exist a renv.so in libs/
This needs to be patched and signed so it may be loaded later on.
before this version 1.0.11 it didn't exist.